package.json

```diff
@@ -75,7 +75,7 @@
     "jwk-to-pem": "^2.0.5",
     "knex": "^2.4.2",
     "mozlog": "^3.0.2",
-    "next": "^13.4.1",
+    "next": "^13.5.4",
```

Comment:
It is a related change - a recent release suppressed the warning about running server components in the browser even though they were just running in Jest. That said, after rebasing this will disappear, since we've already updated to the latest version in `main` now.